settings.gradle

```diff
@@ -5,13 +5,14 @@ plugins {
 
 rootProject.name = 'spring-functions-catalog'
 
-include 'function-dependencies'
+include 'spring-function-dependencies'
 
 ['common', 'consumer', 'function', 'supplier'].each { group ->
     def groupDir = file(group)
     groupDir.eachDir { subProject ->
         include "${subProject.name}"
-        project(":${subProject.name}").projectDir = new File(groupDir.absolutePath, subProject.name)
+        def project = project(":${subProject.name}")
+        project.projectDir = new File(groupDir.absolutePath, subProject.name)
+        project.name = 'spring-' + project.name
```

Comment:
Why don’t rename all the dirs instead to make it clear obvious?

Reply:
Yep, this was callout **ONE:** in PR desc

> ONE: I did not actually rename the folders of the sub-modules as that leads to bit lengthy, repetitive name. However, I am not strongly opposed to doing that if you think it would be better to keep the folder name in sync w/ the artifiactId.

Also, please take a peek at callout **TWO** when you get a chance.